projects/Flux/onnx/.polyaxonignore

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
This file only needs to be at the top of the repo (or in the repo once). The docs are probably not clear enough, but you should create/init the polyaxon project and the root of the github repo. It'll then create the .polyaxon and .polyaxonignore there.

Reply:
:+1: